todo/src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import rootReducer from './reducers/rootReducer'
+
+ReactDOM.render(
+  <Provider store={createStore(rootReducer)}>
```

Comment:
you may want to consider breaking createstore out of provider, and using a const store = createstore()  then you can pass in your reducer, as well as the redux devtools explained in video 2.  Let me know if you need any help setting that up.